Sources/CoherentSwiftCore/Helpers/FileScanner.swift

```diff
@@ -24,21 +24,25 @@ public class FileScanner {
     }
 
     public func parse(with configuration: Configuration, parentPath: Path) throws {
-        let absoluteSourcePath = try parentPath.safeJoin(path: configuration.sourcePath())
-        guard absoluteSourcePath.exists else {
-            throw RuntimeError("Source folder '\(absoluteSourcePath.string)' doesn't exist")
+        let absoluteSourcePaths = configuration.sourcePaths()
+            .flatMap { $0.glob("") }
+            .compactMap { try? parentPath.safeJoin(path: $0) }
+            .filter { $0.exists }
+
+        guard !absoluteSourcePaths.isEmpty else {
+            throw RuntimeError("Source folders '\(absoluteSourcePaths)' doesn't exist")
```

Comment:
* This logic needs a refactor. Right now you have only the list of existing sources, so if it's empty the message will be:
```
Source folders '[]' doesn't exist
```

* Can we provide a more meaningful feedback too?
i.e.:

Case |  Outcome
----- | -----
All sources exist | Proceed
None of the sources exist | RuntimeError with generic message
At least one source exists but others don't | RuntimeError specifying the list of sources that don't exist **OR** simply log a warning message with the same content*

\* This could be based on a flag